### PR TITLE
Try a more informative title

### PR DESCRIPTION
**Pull request by leouieda**
Continuing from #1, this is a stab at a better title.

**Reminders**:

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst` and the base `__init__.py` file for the package.
- [ ] Write detailed docstrings for all functions/classes/methods. It often helps to design better code if you write the docstrings first.
- [ ] If adding new functionality, add an example to the docstring, gallery, and/or tutorials.
- [ ] Add your full name, affiliation, and [ORCID](https://orcid.org) (optional) to the `AUTHORS.md` file (if you haven't already) in case you'd like to be listed as an author on the [Zenodo](https://zenodo.org/communities/fatiando) archive of the next release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
-# Lessons learned from the past 11 years developing the Fatiando a Terra project
+# Design useful tools that do one thing well and work together: rediscovering the UNIX philosophy while building the Fatiando a Terra project
 
-Fatiando a Terra Project,
 Leonardo Uieda<sup>1</sup>,
-ADD YOUR NAME AND AFFILIATION HERE
+ADD YOUR NAME AND AFFILIATION HERE,
```
Comment (LL-Geo) [inline README.md:4]: My name and affiliation:
Lu Li
School of Earth Sciences, The University of Western Australia, Australia

**Review by LL-Geo — APPROVED** [review]
I love the new title here 😄

**Comment by santisoler** [comment]
I really like this title! 

Unix ❤️

File: README.md (hunk 1 above)
Comment (santisoler) [reply README.md:4]: I opened #3 for adding the authors, I already included your name and affiliation. Thanks @LL-Geo !

**Review by santisoler — APPROVED** [review]

**Pull request merged**